app/components/bali/carousel/component.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Bali
+  module Carousel
+    class Component < ApplicationViewComponent
+      renders_many :images
```

Comment:
Hay que renombrarlo a items, porque realmente puedes poner cualquier cosa dentro del carousel, no solo imágenes

```suggestion
      renders_many :items
```